mmedit/models/extractors/lte.py

```diff
@@ -0,0 +1,97 @@
+import torch
+import torch.nn as nn
+from mmcv.runner import load_checkpoint
+from torchvision import models
+
+from mmedit.models import ImgNormalize
+from mmedit.models.registry import COMPONENTS
+from mmedit.utils import get_root_logger
+
+
+@COMPONENTS.register_module()
+class LTE(nn.Module):
+    """Learnable Texture Extractor
+
+    Based on pretrained VGG19. Generate features in 3 levels.
+
+    Args:
+        requires_grad (bool): Require grad or not. Default: True.
+        pixel_range (float): Pixel range of geature. Default: 1.
+        pretrained (str): Path for pretrained model. Default: None.
+    """
+
+    def __init__(self, requires_grad=True, pixel_range=1., pretrained=None):
+        super().__init__()
+
+        vgg_mean = (0.485, 0.456, 0.406)
+        vgg_std = (0.229 * pixel_range, 0.224 * pixel_range,
+                   0.225 * pixel_range)
+        self.sub_mean = ImgNormalize(
```

Comment:
sub_mean -> img_normalize

It deals with both mean and std

Reply:
Will be changed to self.normalize